reference-stores/next/pages/_app.js

```diff
@@ -28,12 +28,14 @@ const MyApp = ({ Component, pageProps, products, components }) => {
 };
 
 MyApp.getInitialProps = async (appContext) => {
-  const data = await nacelleClient.query({ query: SITE_QUERY }).then((site) => {
-    return resolveSiteData({ client: nacelleClient, site });
-  });
+  const { data } = await nacelleClient
+    .query({ query: SITE_QUERY })
+    .then((site) => {
+      return resolveSiteData({ client: nacelleClient, site });
+    });
```

Comment:
An implicit return might be nice here

```suggestion
    .then((site) => resolveSiteData({ client: nacelleClient, site }));
```